server/aws/mc-variables.tf

```diff
@@ -0,0 +1,85 @@
+##
+#  Mectrics Collection Variables File mc-
+##
+
+variable "service_name" {
+  type        = string
+  description = "Name of the service"
+  default     = "cds-covid-alert-create-metric-record"
+}
+
```

Comment:
The default value and parameterization of the lambda was to genericize the lambda.tf so it could be reused by simply attaching a new variable.tf file. Time saver for future work.